src/CodeGenerator/GeneratorTrait.php

```diff
@@ -80,4 +80,12 @@ public function setOutputDirectory(string $dir, ?int $mode = null) : self
 
         return $this;
     }
+
+    /**
+     * @return string
+     */
+    public function getOutputDirectory(): ?string
```

Comment:
Not sure where it is documented, but in zf libraries we are using single space around ":".
It will be checked and fixed with new zend-coding-standard

Reply:
I'll check this with the latest version of the coding standards, thanks.

Reply:
@tux-rampage check this out: https://github.com/zendframework/zend-expressive/pull/530#discussion_r154135280